build/pkgs/pyzmq/patches/0001-setup.py-Remove-setting-of-rpath.patch

```diff
@@ -0,0 +1,25 @@
+From 29427869ce0a9f13e29c7f89873a1880c8be55a1 Mon Sep 17 00:00:00 2001
+From: Matthias Koeppe <mkoeppe@math.ucdavis.edu>
+Date: Sun, 5 Nov 2023 21:12:48 -0800
+Subject: [PATCH 1/2] setup.py: Remove setting of rpath
+
+---
+ setup.py | 2 --
+ 1 file changed, 2 deletions(-)
+
+diff --git a/setup.py b/setup.py
+index d5c77a23..8a2a4943 100755
+--- a/setup.py
++++ b/setup.py
+@@ -300,8 +300,6 @@ def settings_from_prefix(prefix=None):
+                     settings['include_dirs'] += [pjoin(env, 'include')]
+                     settings['library_dirs'] += [pjoin(env, 'lib')]
+ 
+-        for path in settings['library_dirs']:
+-            _add_rpath(settings, os.path.abspath(path))
+     info(settings)
+ 
+     return settings
+-- 
+2.42.0
```

Comment:
I'm curious: is this the `git` version number?

Reply:
Yes; `git format-patch` puts it there, I'm not sure for what exact purpose